MohammadHajjaj/pokemon-go-app
# Add initial Prisma schema setup and implement seed logic

### SUMMARY
create the initial setup for the Prisma schema, the logic for seeding the database from the provided Pokémon .xlsx data, and a Docker compose configuration for PostgreSQL.

the data to be seeded will be loaded from the .xlsx file proivded in the prisma/seed folder
updating existing columns should not result in any issue while seeding the data
if any new colums is added in the .xlsx file, the columns should be added to the prisma schema and `columnsMappings.ts` file
 ### TEST PLAN
-  make sure that the database has been setup correctly and the migrations have been applied
-  run the seed file locally and made sure that the database is seeded with the correct data
- try to update the data to be seeded, to make sure that the seed file won't break if any of the values has changed
